builder/entitlements.mac.plist

```diff
@@ -4,5 +4,7 @@
   <dict>
     <key>com.apple.security.cs.allow-unsigned-executable-memory</key>
     <true/>
+    <key>com.apple.security.cs.disable-library-validation</key>
```

Comment:
What are the implications of this configuration key?

Reply:
As far as I can tell, this prevents macOS from running binaries with unsigned linked libraries packaged within a DMG.
Setting this config key to `false` disables such checks.

[Here's some more info on this config key](https://developer.apple.com/documentation/bundleresources/entitlements/com_apple_security_cs_disable-library-validation).

For some reason this error only pops up in the `xla/1236-thank-you-next` branch, I assume that `openssl` wasn't linked in `proxy` before?

On my local dev machine as well as on CI `openssl` was installed via brew:
```
~ » brew info openssl
openssl@1.1: stable 1.1.1i (bottled) [keg-only]
Cryptography and SSL/TLS Toolkit
...
For compilers to find openssl@1.1 you may need to set:
  export LDFLAGS="-L/usr/local/opt/openssl@1.1/lib"
  export CPPFLAGS="-I/usr/local/opt/openssl@1.1/include"
```

And it is not signed by anyone.
```
~ » codesign -dv --verbose=4  /usr/local/opt/openssl@1.1/lib/libssl.1.1.dylib
/usr/local/opt/openssl@1.1/lib/libssl.1.1.dylib: code object is not signed at all
```